webpack/assets/javascripts/react_app/redux/actions/status/index.js

```diff
@@ -0,0 +1,12 @@
+import { STATUS_REQUEST, STATUS_SUCCESS, STATUS_FAILURE } from '../../consts';
```

Comment:
shouldnt these const be more specific? e.g. HOST_STATUS or COMPUTE_RESOURCE_STATUS?

Reply:
I've extracted it out of the tree, something like this:

STATUS
    -COMPUTE_RESOURCE
        [-X...] 
    -SMART_PROXY
        [-Y...]